apps/web/src/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationActions.tsx

```diff
@@ -122,6 +172,163 @@ const SupportAction = ({ show }: SupportActionProps) => {
+            )
+          : intl.formatMessage(
+              {
+                defaultMessage: "Your {title} application's ID copied",
+                id: "ftg8sT",
+                description:
+                  "Button text to indicate that a specific application's ID has been copied",
+              },
+              {
+                title: jobTitle,
+              },
+            )
+      }
+    >
+      {linkCopied
+        ? intl.formatMessage({
+            defaultMessage: "Application ID copied",
+            id: "gBAz/G",
+            description:
+              "Button text to indicate that a specific application's ID has been copied",
+          })
+        : intl.formatMessage({
+            defaultMessage: "Copy application ID",
+            id: "rvoNoQ",
+            description: "Button text to copy a specific application ID",
+          })}
+    </Button>
+  );
+};
+export interface VisitResumeActionProps extends ActionProps {
+  userID: string;
+  application: Application;
+}
+const VisitResumeAction = ({
+  show,
+  userID,
+  application,
+}: VisitResumeActionProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const jobTitle = getFullPoolTitleHtml(intl, application.pool) && "";
+
+  if (!show) {
+    return null;
+  }
+
+  const recruitmentSectionUrl = paths.resumeAndRecruitment(userID, {
+    applicationId: application.id,
+    section: PAGE_SECTION_ID.QUALIFIED_RECRUITMENT_PROCESSES,
+  });
+
+  return (
+    <Link
+      href={recruitmentSectionUrl}
+      mode="inline"
+      data-h2-color="base(black.light)"
+      data-h2-font-size="base(caption)"
+      aria-label={intl.formatMessage(
+        {
+          defaultMessage: "Visit the {title} recruitment on your résumé",
+          id: "pqsnpk",
+          description: "Link text to direct a user to the Résumé page",
+        },
+        {
+          title: jobTitle,
+        },
+      )}
+    >
+      {intl.formatMessage({
+        defaultMessage: "Visit résumé",
+        id: "Dq+GKf",
+        description: "Link text to direct a user to the Résumé page",
+      })}
+    </Link>
+  );
+};
+
+export interface ManageAvailabilityActionProps extends ActionProps {
+  userID: string;
+  application: Application;
+}
+const ManageAvailabilityAction = ({
+  show,
+  userID,
+  application,
+}: ManageAvailabilityActionProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const jobTitle = getFullPoolTitleHtml(intl, application.pool) && "";
+
+  if (!show) {
+    return null;
+  }
+
+  return (
+    <Link
+      href={paths.profile(userID)}
+      mode="inline"
+      data-h2-color="base(black.light)"
+      data-h2-font-size="base(caption)"
+      aria-label={intl.formatMessage(
+        {
+          defaultMessage:
+            "Manage your availability for the {title} recruitment",
+          id: "3QkRNc",
+          description:
+            "Link text to direct a user to change the availability of the specific recruitment process",
```

Comment:
This says "manage availability" for a specific process, but it links to the profile page where no recruitments are present
Is this perhaps mixed up with the visit resume page that links to recruitments?

Reply:
Its coming in 6772
[ ] - Manage availability, which also only appears on Qualified applications, links you to the Account and privacy settings > recruitment availability section of the Profile (soft dependency on https://github.com/GCTC-NTGC/gc-digital-talent/issues/6772)

Reply:
Ah, got it, nevermind then